lib/ruby_lsp/tapioca/server_addon.rb

```diff
@@ -24,22 +24,38 @@ def execute(request, params)
             halt_upon_load_error: false,
           ).load_dsl_extensions_and_compilers
         when "dsl"
-          fork { dsl(params[:constants]) }
+          fork do
+            with_notification_wrapper("dsl", "Generating DSL RBIs") do
+              dsl(params[:constants])
+              end_progress("dsl")
```

Comment:
Why doesn't the wrapper end it after yielding the block?

Reply:
Not a big deal here but it would be cleaner in the future. Looks easy to forget to do.